mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -71,79 +68,78 @@ You are not restricted to only see your image in 2D. The Output Inspector offers
+Connections between compatible outputs and inputs are established automatically if two modules get close enough to each other.
 
 {{<alert class="info" caption="Extra Infos">}}
-Connecting, Disconnecting, Moving and Replacing Connections is explained {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch03s04.html" "here" >}}
+Connecting, Disconnecting, Moving and Replacing Connections is furtherly explained {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch03s04.html" "here" >}}
 {{</alert>}}
 
 [//]: <> (MVL-653)
 
 ### Image Processing {#TutorialImageProcessing}
 
-Next, we like to add a processing step to our network. We like to smooth the image using an average kernel. In order to do that, search for the module `Convolution` and add it to the workspace. Delete the established connection from the module `ImageLoad` to the module `View2D` by clicking on the connection and pressing {{< keyboard "DEL" >}}. Now, you can build new connections from the module `ImageLoad` to the module `Convolution` and from this module to `View2D`.
+An average kernel will be used to smooth the image out as our next step will be to actually process our image. Add the `Convolution`module to your workspace and disconnect the `View2D`module from the `ImageLoad`module by clicking on the connection and pressing {{< keyboard "DEL" >}}. Now, you can build new connections from the module `ImageLoad` to the module `Convolution` and the `Convolution`module to `View2D`.
 
 ![Convolution Module](/images/tutorials/basicmechanics/BM_08.png "Convolution Module")
 
-Double-click the module `Convolution` to open its panel. The panel allows configurations of the module. You can adjust parameters or in this case select a kernel, for example the *3x3 Avarage Kernel*.
+Open the panel of the `Convolution`module by double-clicking it. The panel allows configuration of the module. You can adjust parameters or select a kernel. We will be using the *3x3 Avarage Kernel* for now.
 
 ![Select a Kernel](/images/tutorials/basicmechanics/BM_09.png "Select a Kernel")
 
 The module `View2D` is now displaying the smoothed image.
 
-To see the difference between the processed and the unprocessed image, click on the output of the module `ImageLoad` to view the original image in the Output Inspector. The great thing about the Output Inspector is that it can display the output of any connector in the process chain (as long as a format is used, the inspector can interpret). Simply click the connector or connection to find out more about the module output.
+To compare the processed and unprocessed image, click on the output connector of the module `ImageLoad` to display the original image in the Output Inspector. The Output Inspectors greatest advantage is, that it's able to display the output of any connector in the process chain (as long as an interpretable format is used). Simply click the connector or connection to find out more about the modules output.
```

Comment:
the modules output -> the module output (or module's output)

Reply:
"module's output" actually is not right, as the apostroph indicates an abbreviated "is". 
Let's compromise on "the module output" :)

Reply:
Nevermind, googled it and you are right. The option I went with is fine though.